### DISCUSSION TITLE
Screen Layout designer

### DISCUSSION THREAD

**Discussion by ghost** [Bodmer/TFT_eSPI#996] [Q&A - General]
Seeing as using TFT_espi allows the use of a lot of different displays with the same code interface, has anyone come up with a simple layout designer?

I'm thinking something that allows the various draw functions to be called without having to recompile and upload a sketch, something like a sketch that translates commands over serial to actual draw functions.

**Comment by Bodmer** [comment]
It appears you are asking for 2 things:
1. An interpretter sketch running on the processor that interprets graphics commands from a PC
2. A layout design program that runs on the PC

I think there would be little demand for such functionality.  The closest I have seen to a PC layout editior is [GUISlice](https://github.com/ImpulseAdventure/GUIslice).
